test/test_pooling.py

```diff
@@ -411,6 +411,41 @@ def find_one():
         # maxConnecting = unbounded: 30+ connections in ~0.140+ seconds
         print(len(pool.conns))
 
+    def test_csot_timeout_message(self):
+        with self.assertRaises(Exception) as error:
+            with timeout(0.001):
+                self.client.db.t.find_one({"$where": delay(1)})
+
+        self.assertTrue("(configured timeouts: timeoutMS: 1.0ms" in str(error.exception))
+
+    def test_socket_timeout_message(self):
+        client = rs_or_single_client(socketTimeoutMS=0.01)
+
+        with self.assertRaises(Exception) as error:
+            client.db.t.find_one({"$where": delay(1)})
+
+        self.assertTrue("(configured timeouts: socketTimeoutMS: 0.01ms" in str(error.exception))
+
+    @client_context.require_failCommand_fail_point
+    def test_connection_timeout_message(self):
+        client = rs_or_single_client(connectTimeoutMS=1)
+
+        # Mock a connection failing due to timeout.
+        mock_connection_timeout = {
```

Comment:
This is failing on the find command, not the connection establishment. Is that expected?

Reply:
@NoahStapp ping.

Reply:
Oh I see test_connection_timeout_message does the right thing now. Resolving.

Reply:
That is outdated, the current code for `test_connection_timeout_message` correctly fails on the connection establishment.